### ISSUE TITLE
Error when testing: Unrecognized arguments: --prototype_num

### ISSUE THREAD

**Issue by BenPashley** [dddavid4real/HistGen#12]
Hi,

The prototype_num parameter is missing when executing training:

usage: main_test_AllinOne.py [-h] [--image_dir IMAGE_DIR] [--ann_path ANN_PATH] [--dataset_name {iu_xray,mimic_cxr,wsi_report}] [--max_seq_length MAX_SEQ_LENGTH] [--threshold THRESHOLD]
                             [--num_workers NUM_WORKERS] [--batch_size BATCH_SIZE] [--model_name {histgen}] [--visual_extractor VISUAL_EXTRACTOR]
                             [--visual_extractor_pretrained VISUAL_EXTRACTOR_PRETRAINED] [--d_model D_MODEL] [--d_ff D_FF] [--d_vf D_VF] [--num_heads NUM_HEADS] [--num_layers NUM_LAYERS]
                             [--dropout DROPOUT] [--logit_layers LOGIT_LAYERS] [--bos_idx BOS_IDX] [--eos_idx EOS_IDX] [--pad_idx PAD_IDX] [--use_bn USE_BN] [--drop_prob_lm DROP_PROB_LM]
                             [--topk TOPK] [--cmm_size CMM_SIZE] [--cmm_dim CMM_DIM] [--region_size REGION_SIZE] [--sample_method SAMPLE_METHOD] [--beam_size BEAM_SIZE]
                             [--temperature TEMPERATURE] [--sample_n SAMPLE_N] [--group_size GROUP_SIZE] [--output_logsoftmax OUTPUT_LOGSOFTMAX] [--decoding_constraint DECODING_CONSTRAINT]
                             [--block_trigrams BLOCK_TRIGRAMS] [--n_gpu N_GPU] [--epochs EPOCHS] [--save_dir SAVE_DIR] [--record_dir RECORD_DIR] [--save_period SAVE_PERIOD]
                             [--monitor_mode {min,max}] [--monitor_metric MONITOR_METRIC] [--early_stop EARLY_STOP] [--log_period LOG_PERIOD] [--optim OPTIM] [--lr_ve LR_VE] [--lr_ed LR_ED]
                             [--weight_decay WEIGHT_DECAY] [--amsgrad AMSGRAD] [--lr_scheduler LR_SCHEDULER] [--step_size STEP_SIZE] [--gamma GAMMA] [--seed SEED] [--resume RESUME]
                             [--load LOAD]
main_test_AllinOne.py: error: unrecognized arguments: --prototype_num 512

**Comment by dddavid4real** [comment]
Thanks for reporting this error. Will have a look.
